app/scripts/metamask-controller.js

```diff
@@ -3320,74 +3213,6 @@ export default class MetamaskController extends EventEmitter {
     await this.threeBoxController.init();
   }
 
-  /**
-   * Sets whether or not to use the blockie identicon format.
-   * @param {boolean} val - True for bockie, false for jazzicon.
-   * @param {Function} cb - A callback function called when complete.
-   */
-  setUseBlockie(val, cb) {
```

Comment:
All of these wrapper functions were no longer needed. These functions were manually attempts at "nodeify-ing" the called functions essentially.